stix_shifter_modules/proxy/configuration/config.json

```diff
@@ -1,20 +1,19 @@
 {
     "connection": {
         "type": {
-            "default": "proxy",
             "displayName": "Proxy"
         },
         "host": {
             "type": "text",
             "regex": "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\\-]*[A-Za-z0-9])$"
         },
         "port": {
-            "default": 443,
-            "type": "number"
+            "type": "number",
+            "default": 443
         },
         "help": {
-            "default": "https://www.ibm.com/support/knowledgecenter/SSTDPP_1.1.0/docs/scp-core/data-sources-proxy.html",
-            "type": "link"
+            "type": "link",
+            "default": "https://www.ibm.com/support/knowledgecenter/SSTDPP_1.1.0/docs/scp-core/data-sources-proxy.html"
```

Comment:
I guess we have no choice for now, but these links wouldn't be accessible to anyone without an IBM or partner login.

Reply:
it makes sense to duplicate that page somewhere on a public resource.